### PR TITLE
Add mysql orm

### PR DESCRIPTION
**Pull request by saaeg657**

### CODE REVIEW

File: src/server/mutation/user.mutation.js [hunk 1]
```diff
@@ -48,22 +55,31 @@ const createUserMutation = {
...
-          id: createdUser.uid,
+      .then(createdUser => refs.user.root.child(createdUser.uid).set({
+        id: createdUser.uid,
+        e,
+        n,
+        pw: bcrypt.hashSync(pw, saltRounds),
+        cAt: Date.now(),
+        ...defaultSchema.user.root
+      })
+        .then(() => refs.user.userQualification.child(createdUser.uid).set({
```
Comment (ochanje210) [inline src/server/mutation/user.mutation.js]: here, indentation should be same as above ```then()``` method

Comment (ochanje210) [inline src/server/mutation/user.mutation.js:64]: is there a specific reason for having 2-depth Promise chain?

File: src/server/mutation/user.mutation.js [hunk 2]
```diff
@@ -224,6 +268,14 @@ const userAddAddressMutation = {
...
         lon
       })
+      // mysql
+      .then(() => createData(mRefs.user.userAddress, {
+        name,
+        mAddr,
+        sAddr,
+        lat,
+        lon
+      }, '5d356c10-3d11-11e7-9ad1-77c411db57c7'))
```
Comment (ochanje210) [inline src/server/mutation/user.mutation.js]: I wonder what this is

File: sequelize.config.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (lyw0149) [inline sequelize.config.json]: this file safe?

File: src/server/mutation/user.mutation.js (hunk 2 above)
Comment (saaeg657) [reply src/server/mutation/user.mutation.js]: Mistake

File: sequelize.config.json (hunk 3 above)
Comment (saaeg657) [reply sequelize.config.json]: This should be deleted.

File: src/server/mutation/user.mutation.js (hunk 1 above)
Comment (saaeg657) [reply src/server/mutation/user.mutation.js:64]: For reference to createdUser?

**Review by lyw0149 — APPROVED** [review]
굿

**Pull request merged**